kubernetes/linera-validator/templates/server.yaml

```diff
@@ -60,4 +60,4 @@ spec:
             - name: RUST_LOG
               value: {{ .Values.logLevel }}
             - name: RUST_BACKTRACE
-              value: "1"
+              value: "1"
```

Comment:
Please can we have newlines at the end of test files?

Reply:
My bad, not sure why it got removed